libs/tools/barista/src/builder/strapi.ts

```diff
@@ -76,12 +78,31 @@ export const strapiBuilder: BaPageBuilder = async (
         environment.strapiEndpoint,
       );
 
+  let categoriesData: BaStrapiCategory[] = [];
+
+  categoriesData = await fetchContentList<BaStrapiCategory>(
+    BaStrapiContentType.Categories,
+    { publicContent: isPublicBuild() },
+    environment.strapiEndpoint,
+  );
+
+  categoriesData = next
+    ? categoriesData.filter((data) => data.nextpages.length > 0)
+    : categoriesData.filter((data) => data.pages.length > 0);
+
   // Filter pages with draft set to null or false
   pagesData = pagesData.filter((page) => !page.draft);
 
+  // Results from transforming the pagecontent
   const transformed: BaPageBuildResult[] = [];
 
+  // Array of categories for corresponding design systems
+  const categories: string[] = [];
+
   for (const page of pagesData) {
+    if (next && page.category === null) {
+      categories.push(page.title);
+    }
```

Comment:
could you add a comment here please - why if there is no category we use the title instead. Maybe use an example to outline it more clearly.

Reply:
@rowa-audil Please use next time for comments inside the code the `//` annotation and not the jsDoc. JsDoc is used to annotate a method, property, etc but for code comments always use `//` as there is a difference on cleaning up the code.